services/activities.js

```diff
@@ -60,3 +60,23 @@ exports.addActivity = async (data) => {
     throw Error(err.message)
   }
 }
+exports.updateActivity = async (id, name, image, content) => {
+  try {
+    const activity = await Activity.findByPk(id)
+
+    if (!activity) {
+      throw Error(`ID ${id} not found.`)
+    }
+    activity.set({
+      id, name, image, content,
+    })
+    activity.save()
+
+    return activity
+  } catch (error) {
+    if (error.message === 'ID not found.') {
```

Comment:
el if no va a coincidir porque el mensaje de error es distinto.  trae el id.

                 throw Error(`ID ${id} not found.`)
if (error.message === 'ID not found.') {